test/externalTests/solc-js/index.js

```diff
@@ -0,0 +1,13 @@
+const semver = require('semver');
+
+require('./linker.js');
+require('./translate.js');
+require('./package.js');
+require('./smtchecker.js');
+require('./abi.js');
+require('./determinism.js');
```

Comment:
We can just inject this line instead of copying the whole file

Reply:
Done.